### PR TITLE
Remove references to Optimizely

### PR DESCRIPTION
**Pull request by maureensturgeon**
First iteration of removing references to optimizely. There is also a reference in a CSV for marketing that needs to be updated and I want to wait until the I18n sync happens and all remaining references are removed before I remove it from the ignore list of the link checker (just incase).

## Links

- jira ticket: [LP-2250](https://codedotorg.atlassian.net/browse/LP-2250)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/educate/it.md [hunk 1]
```diff
@@ -65,15 +65,14 @@ theme: responsive
...
+| ---------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | **Required**                                                                             | `https://code.org`<br/>`https://*.code.org`                                                                                                            |
-| **Optional**                                                                             | `https://cdn.optimizely.com`<br/>`https://www.google-analytics.com`<br/>`https://www.googleoptimize.com`                                               |
+| **Optional**                                                                             | `https://www.google-analytics.com`<br/>`https://www.googleoptimize.com`                                                                                |
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/code.org/public/educate/it.md:75]: tiny nit: is the extra whitespace needed?

**Review by bethanyaconnor — APPROVED** [review]

File: pegasus/sites.v3/code.org/public/educate/it.md (hunk 1 above)
Comment (jamescodeorg) [reply pegasus/sites.v3/code.org/public/educate/it.md:75]: This support page will also need to be updated: https://support.code.org/hc/en-us/articles/202591743-Technical-requirements-for-Code-org

Comment (maureensturgeon) [reply pegasus/sites.v3/code.org/public/educate/it.md:75]: Oh, I was thinking that was being generated from this file, but now I realize I think it's a different one. Will update.

Comment (jamescodeorg) [reply pegasus/sites.v3/code.org/public/educate/it.md:75]: Yeah, that page needs to be manually updated in zendesk.  I just remembered that Josh said that there are also versions of this support page in other languages (that he updated for me when I made the google optimize change).

Comment (maureensturgeon) [reply pegasus/sites.v3/code.org/public/educate/it.md:75]: @bethanyaconnor the table auto-formatted for me on save.. not sure if it has to do with the width of the table or what

Comment (maureensturgeon) [reply pegasus/sites.v3/code.org/public/educate/it.md:75]: @jamescodeorg Oh I just made a change to [i18n/locales/source/markdown/public/educate/it.md](https://github.com/code-dot-org/code-dot-org/pull/47026/files#diff-1e226bf3f4ac700054ac70c65bcb638b25b51f3048bb7a9354db9484510bb135) is that not right?

**Pull request merged**